src/components/Footer/index.tsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react'
+import Link from 'next/link'
 
-const Footer = () => {
-  return <div className="footer-container">Footer</div>
-}
+const Footer = () => (
+  <div className="footer">
+    Footer
+    <div className="footer__item">
+      <Link href="/about-us">
+        <a className="footer__link">About Us</a>
```

Comment:
I don't see a class for `footer__link` in `_footer.scss`?

Reply:
Either remove it or add an empty class there and put a comment(`// TODO: For future implementation`).